### DISCUSSION TITLE
Q: 我添加一个表情包能显示，多个就不行了是我的设置有问题吗？我通过自己的样式修改不了表情包大小吗？因该怎么设置。 || Q: When I add an emoticon, it can be displayed, but multiple emoticons cannot be displayed. Is there something wrong with my settings? Can't I modify the size of the emoticon package through my own style? How to set it up.

### DISCUSSION THREAD

**Discussion by yeminxi** [walinejs/waline#2836] [Q&A] [answered]
### Describe the bug | 问题描述

![image](https://github.com/user-attachments/assets/9fd5d877-a962-4536-aef8-8a03ba115ebf)
![image](https://github.com/user-attachments/assets/5c99d6d2-2065-443b-8df1-5f970afc5494)
![image](https://github.com/user-attachments/assets/f7ab5e11-37c8-4af4-8b74-f5368941e4f1)

### Website URL | 问题网站

https://ymxblog.us.kg

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

![image](https://github.com/user-attachments/assets/9fd5d877-a962-4536-aef8-8a03ba115ebf)
![image](https://github.com/user-attachments/assets/5c99d6d2-2065-443b-8df1-5f970afc5494)
![image](https://github.com/user-attachments/assets/f7ab5e11-37c8-4af4-8b74-f5368941e4f1)

### Website URL | Problem website

https://ymxblog.us.kg

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is the data stored?

LeanCloud(https://leancloud.app)

**Comment by Zengzzhao** [comment, marked as answer]
> ### Describe the bug | 问题描述
> ![image](https://private-user-images.githubusercontent.com/169566216/391315079-9fd5d877-a962-4536-aef8-8a03ba115ebf.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzY5MTkzNDEsIm5iZiI6MTczNjkxOTA0MSwicGF0aCI6Ii8xNjk1NjYyMTYvMzkxMzE1MDc5LTlmZDVkODc3LWE5NjItNDUzNi1hZWY4LThhMDNiYTExNWViZi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMTE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDExNVQwNTMwNDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00OWY4MTg0OWEwYWEyZTk2MmNmOWQ0ODlhOGFlZTk3NTA4MWJjMDg4YjQ5YmFmNmM2YTUyZWY1ZDAzMjVmMDNmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.qxmxRLzsIiOw7rRnNAQO8wWRk_5QTpa708dgZXd7Qyo) ![image](https://private-user-images.githubusercontent.com/169566216/391315163-5c99d6d2-2065-443b-8df1-5f970afc5494.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzY5MTkzNDEsIm5iZiI6MTczNjkxOTA0MSwicGF0aCI6Ii8xNjk1NjYyMTYvMzkxMzE1MTYzLTVjOTlkNmQyLTIwNjUtNDQzYi04ZGYxLTVmOTcwYWZjNTQ5NC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMTE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDExNVQwNTMwNDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03MTAzMjc4NjRkMzNlMGY5ODM4MTMyOGI4ODkzYjJlZTZmODIyMDJhMGQxNDBiM2YwMDBmYWUyNzZkNGMxYTlmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.AxoLE56YD7hsKynDVyZ3EOOYVzJDcvKmg5MG_xgT0Z0) ![image](https://private-user-images.githubusercontent.com/169566216/391315277-f7ab5e11-37c8-4af4-8b74-f5368941e4f1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzY5MTkzNDEsIm5iZiI6MTczNjkxOTA0MSwicGF0aCI6Ii8xNjk1NjYyMTYvMzkxMzE1Mjc3LWY3YWI1ZTExLTM3YzgtNGFmNC04Yjc0LWY1MzY4OTQxZTRmMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMTE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDExNVQwNTMwNDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jMTgxMzU1NmUxZGE3NTEwMDVjNzc5MjAyYTgxZDMxM2YxN2NlNzgxYTY2YTZhNTAzMTc0MzkxYjc1OGUzODA3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.uBO8qFrg5zwyyCmzN87z1lT1dYQMTtBrwPRsPaEjQ0k)
> 
> ### Website URL | 问题网站
> https://ymxblog.us.kg
> 
> ### Where your waline deploy? | 服务部署在哪里？
> Vercel (Default)
> 
> ### Where your comment data store? | 数据存储在哪里？
> LeanCloud(https://leancloud.app)
> 
> ### Describe the bug | Problem description
> ![image](https://private-user-images.githubusercontent.com/169566216/391315079-9fd5d877-a962-4536-aef8-8a03ba115ebf.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzY5MTkzNDEsIm5iZiI6MTczNjkxOTA0MSwicGF0aCI6Ii8xNjk1NjYyMTYvMzkxMzE1MDc5LTlmZDVkODc3LWE5NjItNDUzNi1hZWY4LThhMDNiYTExNWViZi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMTE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDExNVQwNTMwNDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00OWY4MTg0OWEwYWEyZTk2MmNmOWQ0ODlhOGFlZTk3NTA4MWJjMDg4YjQ5YmFmNmM2YTUyZWY1ZDAzMjVmMDNmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.qxmxRLzsIiOw7rRnNAQO8wWRk_5QTpa708dgZXd7Qyo) ![image](https://private-user-images.githubusercontent.com/169566216/391315163-5c99d6d2-2065-443b-8df1-5f970afc5494.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzY5MTkzNDEsIm5iZiI6MTczNjkxOTA0MSwicGF0aCI6Ii8xNjk1NjYyMTYvMzkxMzE1MTYzLTVjOTlkNmQyLTIwNjUtNDQzYi04ZGYxLTVmOTcwYWZjNTQ5NC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMTE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDExNVQwNTMwNDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03MTAzMjc4NjRkMzNlMGY5ODM4MTMyOGI4ODkzYjJlZTZmODIyMDJhMGQxNDBiM2YwMDBmYWUyNzZkNGMxYTlmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.AxoLE56YD7hsKynDVyZ3EOOYVzJDcvKmg5MG_xgT0Z0) ![image](https://private-user-images.githubusercontent.com/169566216/391315277-f7ab5e11-37c8-4af4-8b74-f5368941e4f1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzY5MTkzNDEsIm5iZiI6MTczNjkxOTA0MSwicGF0aCI6Ii8xNjk1NjYyMTYvMzkxMzE1Mjc3LWY3YWI1ZTExLTM3YzgtNGFmNC04Yjc0LWY1MzY4OTQxZTRmMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMTE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDExNVQwNTMwNDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jMTgxMzU1NmUxZGE3NTEwMDVjNzc5MjAyYTgxZDMxM2YxN2NlNzgxYTY2YTZhNTAzMTc0MzkxYjc1OGUzODA3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.uBO8qFrg5zwyyCmzN87z1lT1dYQMTtBrwPRsPaEjQ0k)
> 
> ### Website URL | Problem website
> https://ymxblog.us.kg
> 
> ### Where your waline deploy? | Where is the service deployed?
> Vercel (Default)
> 
> ### Where your comment data store? | Where is the data stored?
> LeanCloud(https://leancloud.app)

我这加了emoji配置使用官方的例子也不行
